<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lekarze</class>
 <widget class="QWidget" name="lekarze">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lekarze</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tWgNazwisk">
       <attribute name="title">
        <string>Wg nazwisk</string>
       </attribute>
       <widget class="QTableWidget" name="tWgNazwisk_table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>511</width>
          <height>281</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>21</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Wyszukaj:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="tWgNazwisk_filtr">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>0</y>
          <width>291</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tWgNazwisk_filtr_czysc">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>0</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Czyść</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tWgNazwisk_btnNowy">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>0</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Nowy</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tWgGodzinPrzyjec">
       <attribute name="title">
        <string>Wg godzin przyjęć</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dzień tygodnia:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="tWgGodzinPrzyjec_dzien">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>121</width>
          <height>22</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Poniedziałek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wtorek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Środa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czwartek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Piątek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sobota</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Niedziela</string>
         </property>
        </item>
       </widget>
       <widget class="QTableWidget" name="tWgGodzinPrzyjec_table">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>501</width>
          <height>271</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="rowCount">
         <number>2</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>15</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <row/>
        <row/>
        <column/>
        <column/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
